src/Illuminate/Console/Scheduling/ScheduleWorkCommand.php

```diff
@@ -44,7 +44,7 @@ public function handle()
     {
         $this->components->info('Running schedule tasks every minute.');
 
-        [$lastExecutionStartedAt, $keyOfLastExecutionWithOutput, $executions] = [null, null, []];
```

Comment:
looks like this variable was refactored out here:

https://github.com/laravel/framework/pull/43065/files#diff-22402fc2f7976884b0a41ffe7c71d9ef9cf41092ea28cf2897d820b9fa0da8fa